src/components/FindPark.vue

```diff
@@ -30,15 +30,18 @@
 
 <script>
 import { getResults } from '../apiCalls.js';
+import { getSearch } from '../apiCalls.js';
 export default {
   data() {
     return {
       searchTerm: '',
     };
   },
   methods: {
-    search() {
-      //functionality for search goes here...
+    async search() {
+      const results = await getSearch(this.searchTerm)
+        .then((data) => data);
```

Comment:
What's happening with line 43?  Where is the data being stored? Not sure if this is just some shorthand code I'm not used to seeing!

Reply:
Actually, it looks like you might not need line 43 since line 44 handles the trigger of the mutation with an argument of `results`.  Should line 43's `.then( )` be removed?

Reply:
I think you're right in thinking it's unnecessary. It's just taking in the data and returning it... I think it might be left over from when I was doing some `console.logs`.
I'm going to take it out and spin it up locally to see, and I'll make a commit to remove it if it works!

Reply:
Yep, it was redundant, which means so was line 49. Updated the code to remove those unnecessary `.then()`s. Should be good to go, but please review!

Reply:
Awesome.  Thanks, Bret!